nautobot_ssot/integrations/servicenow/models.py

```diff
@@ -0,0 +1,42 @@
+"""Configuration data model for nautobot-ssot-servicenow."""
+
+from django.db import models
+from django.shortcuts import reverse
+
+from nautobot.core.models import BaseModel
+
+
+class SSOTServiceNowConfig(BaseModel):
```

Comment:
IIRC this plugin predates the introduction of `django-constance` into Nautobot core. It might be worth considering whether constance might be a better pattern for configuring this integration - non blocking.